src/test/resources/net/pms/util/prettified_filenames_metadata.json

```diff
@@ -55,15 +55,14 @@
 		}
 	},
 	{
-		"comment": "Video of an extended TV episode",
+		"comment": "Video of a TV episode, noting that EXTENDED is the group and not the edition",
```

Comment:
Is this comment correct? I think the group is usually like `Universal.Media.Server.S01E02.720p-EXTENDED.mkv`

Reply:
That was my original thought, but then the extraction code is not working properly because EXTENDED is not extracted, it matches COMMON_FILE_ENDS_CASE_SENSITIVE and is removed in line 570.

This is what I mean with "Amends one of the testcases so that they correctly
reflect the current working of the extraction code" in the commit message.

Reply:
Yeah I think you are right that it is a bug in the code in that case. I have identified other bugs while writing tests before too, it is one of the many great things about writing tests :)
Maybe we should remove that assertion from the file, and create a new issue for the bug?

Reply:
How about marking some of the items in the JSON as "todo" or s'thing similar? The test code will not assert those, but log success or failure. It will be useful while toying with the regexps to be sure that nothing that used to work broke, while also getting reported what didn't work and works now.

Reply:
I commited so you can see where this is going. It seems to be working, but it's not thoroughly tested yet.